web/models.py

```
@@ -2026,58 +2029,48 @@ class Quiz(models.Model):
+    QUESTION_TYPES = [
+        ("multiple", "Multiple Choice"),
+        ("true_false", "True/False"),
+        ("short", "Short Answer"),
+        ("fill_blank", "Fill in the Blanks"),
+        ("open_ended", "Open-Ended Questions"),
+        ("matching", "Matching Questions"),
+        ("problem_solving", "Problem-Solving Questions"),
+        ("scenario", "Scenario-Based Questions"),
+        ("diagram", "Diagram-Based Questions"),
+        ("coding", "Coding Questions"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)